### PR TITLE
Adding option to pass  publish target

### PR DESCRIPTION
**Pull request by jbosse**
See https://developer.chrome.com/webstore/using_webstore_api#trustedtesters

### CODE REVIEW

**Comment by jbosse** [comment]
I will have to figure out what I did wrong. :-(

File: src/main.ts [hunk 1]
```diff
@@ -61,16 +61,17 @@ async function run() {
     const refresh = core.getInput('refresh-token', { required: true });
     const zip = core.getInput('zip', { required: true });
     const extension = core.getInput('extension');
+    const publishTarget = core.getInput('publish-target') || "default";
```
Comment (trmcnvn) [inline src/main.ts]: You need to define `publish-target` in the `action.yml` file and set the default value there.

**Comment by jbosse** [comment]
Thanks for the feedback!

-Jimmy

**Comment by jbosse** [comment]
I have implemented the missing `action.yml` change and cleaned up the whole PR to just include the files I really changed. Is there a reason to have `node_modules` in the repo?

File: src/main.ts (hunk 1 above)
Comment (trmcnvn) [reply src/main.ts:64]: You can provide a default in the action.yml file

```yaml
default: default
```

**Comment by trmcnvn** [comment]
> I have implemented the missing `action.yml` change and cleaned up the whole PR to just include the files I really changed. Is there a reason to have `node_modules` in the repo?

Yes, the `node_modules` folder is required for the action to be used as a workflow won't install dependencies itself.

**Comment by jbosse** [comment]
Thanks for being so patient.

**Comment by trmcnvn** [comment]
Thanks! 👍

**Pull request merged**